app/design-system/design-system-hedvig/src/main/kotlin/com/hedvig/android/design/system/hedvig/AddonBanner.kt

```diff
@@ -26,7 +26,7 @@ fun FeatureAddonBanner(
   onButtonClick: () -> Unit,
   modifier: Modifier = Modifier,
 ) {
-  val containerColor = HedvigTheme.colorScheme.fillNegative
+  val containerColor = HedvigTheme.colorScheme.borderPrimary
```

Comment:
I know this is up to spec, but using the border color as a container color is a bit sus.
I'll bring this up with design just to see if this is temporary or not.

Reply:
https://hedviginsurance.slack.com/archives/C03U9C6Q7TP/p1737721490406599